# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/aalib/hotfix-return.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- aalib-1.4.0/src/aalinuxkbd.c.vanilla	2025-01-18 17:16:38.289999994 +0100
+++ aalib-1.4.0/src/aalinuxkbd.c	2025-01-18 17:18:10.853333334 +0100
@@ -200,7 +201,7 @@
 static int rawmode_init(void)
 {
     if (!closed)
-	return;
+	return (0);
     mypid = getpid();
     if (tty_fd == -1) {
 	tty_fd = fileno(stdin);
